temporalcli/commands.workflow.go

```diff
@@ -91,7 +121,7 @@ func (c *TemporalWorkflowSignalCommand) run(cctx *CommandContext, args []string)
 			return fmt.Errorf("failed signalling workflow: %w", err)
 		}
 		cctx.Printer.Println("Signal workflow succeeded")
-	} else if batchReq != nil {
+	} else { // batchReq != nil
```

Comment:
Seems fine to me. There was a time where we didn't return `err` when people declined the prompt, so technically all three results of `workflowExecOrBatch` could have been nil, but we changed that and didn't come back and change this, so all good.

Reply:
changed it on delete, cancel, and terminate as well to match